Evergreen/Manifests/WinFSP.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Name": "WinFSP",
```

Comment:
Please rename to `NavimaticsWinFSP` and update this name value to `Navimatics WinFSP` - it's not likely we'll have a second app named `WinFSP`, but ideally apps should have the vendor name in the app name where possible